doc/doxygen/doxyfile.in

```diff
@@ -42,11 +43,14 @@ OPTIMIZE_FOR_FORTRAN   = NO
 OPTIMIZE_OUTPUT_VHDL   = NO
 EXTENSION_MAPPING      =
 MARKDOWN_SUPPORT       = YES
+TOC_INCLUDE_HEADINGS   = 5
```

Comment:
[default value: 5](http://www.doxygen.nl/manual/config.html#cfg_toc_include_headings). Same with autolink support. What are the benefits of explicitly declaring the default values?

Reply:
Again autogeneration. Can't check if it changes anything, the LaTeX docs don't compile. I don't think there's another TOC anywhere else to check, correct me if I'm wrong.

Reply:
I resolved this after you mentioned autogen in the prev comment 😄